pkg/status/render/templates/collector.tmpl

```diff
@@ -29,20 +29,25 @@ Collector
     {{printDashes $CheckName "-"}}{{- if $version }}{{printDashes $version "-"}}---{{ end }}
     {{- range $instance := $CheckInstances }}
       Instance ID: {{.CheckID}} {{status .}}
+      Long Running Check: {{.LongRunning}}
       Configuration Source: {{.CheckConfigSource}}
+      {{- if not .LongRunning }}
       Total Runs: {{humanize .TotalRuns}}
-      Metric Samples: Last Run: {{humanize .MetricSamples}}, Total: {{humanize .TotalMetricSamples}}
-      Events: Last Run: {{humanize .Events}}, Total: {{humanize .TotalEvents}}
+      {{- end }}
+      Metric Samples:{{ if not .LongRunning }} Last Run: {{humanize .MetricSamples}},{{- end }} Total: {{humanize .TotalMetricSamples}}
+      Events:{{- if not .LongRunning }} Last Run: {{humanize .Events}},{{- end }} Total: {{humanize .TotalEvents}}
       {{- range $k, $v := .TotalEventPlatformEvents }}
-      {{ $k }}: Last Run: {{humanize (index $instance.EventPlatformEvents $k) }}, Total: {{humanize $v}}
+      {{ $k }}:{{- if not $instance.LongRunning }}Last Run: {{humanize (index $instance.EventPlatformEvents $k) }},{{- end }} Total: {{humanize $v}}
       {{- end }}
-      Service Checks: Last Run: {{humanize .ServiceChecks}}, Total: {{humanize .TotalServiceChecks}}
+      Service Checks:{{- if not $instance.LongRunning }} Last Run: {{humanize .ServiceChecks}},{{- end }} Total: {{humanize .TotalServiceChecks}}
       {{- if .TotalHistogramBuckets}}
-      Histogram Buckets: Last Run: {{humanize .HistogramBuckets}}, Total: {{humanize .TotalHistogramBuckets}}
+      Histogram Buckets:{{- if not $instance.LongRunning }}Last Run: {{humanize .HistogramBuckets}},{{- end }} Total: {{humanize .TotalHistogramBuckets}}
       {{- end }}
+      {{- if not .LongRunning }}
       Average Execution Time : {{humanizeDuration .AverageExecutionTime "ms"}}
       Last Execution Date : {{formatUnixTime .UpdateTimestamp}}
       Last Successful Execution Date : {{ if .LastSuccessDate }}{{formatUnixTime .LastSuccessDate}}{{ else }}Never{{ end }}
+      {{- end }}
```

Comment:
for code clarity, I'm wondering if we could create 2 sub templates, so it ca be easier to see the result for `long running` or `not long running` check

Reply:
I don't know about subtemplates but I wonder if it could be cleaner to separate both cases with `if` conditions, similar to
```
{{ if .LongRunning }}
Instance ID: ...
Metric Samples: Total: 1313
Long Running Check: true
{{ else }}
Instance ID: ...
Metric Samples: Last Run: 32, Total: 1313
Long Running Check: false
{{ end }}

```

Reply:
some link

```golang
{{ define "longRunningCheckStats" }}
Instance ID: {{.CheckID}} {{status .}}
Configuration Source: {{.CheckConfigSource}}
Long Running Check: true
Total Metric Samples: {{humanize .TotalMetricSamples}}
Total Events: {{humanize .TotalEvents}}
{{- range $k, $v := .TotalEventPlatformEvents }}
{{ $k }} Total: {{humanize $v}}
{{- end }}
Total Service Checks: {{humanize .TotalServiceChecks}}
{{- if .TotalHistogramBuckets}}
Total Histogram Buckets: {{humanize .TotalHistogramBuckets}}
{{- end }}
{{ end }}


{{ define "checkStats" }}
Instance ID: {{.CheckID}} {{status .}}
Configuration Source: {{.CheckConfigSource}}
Metric Samples: Last Run: {{humanize .MetricSamples}}, Total: {{humanize .TotalMetricSamples}}
Events: Last Run: {{humanize .Events}}, Total: {{humanize .TotalEvents}}
{{- range $k, $v := .TotalEventPlatformEvents }}
{{ $k }}: Last Run: {{humanize (index $instance.EventPlatformEvents $k) }}, Total: {{humanize $v}}
{{- end }}
Service Checks: Last Run: {{humanize .ServiceChecks}}, Total: {{humanize .TotalServiceChecks}}
{{- if .TotalHistogramBuckets}}
Histogram Buckets: Last Run: {{humanize .HistogramBuckets}}, Total: {{humanize .TotalHistogramBuckets}}
{{- end }}
Average Execution Time : {{humanizeDuration .AverageExecutionTime "ms"}}
Last Execution Date : {{formatUnixTime .UpdateTimestamp}}
Last Successful Execution Date : {{ if .LastSuccessDate }}{{formatUnixTime .LastSuccessDate}}{{ else }}Never{{ end }}
{{ end }}
```

and then 

``` golang
{{- range $instance := $CheckInstances }}
{{- if not .LongRunning }}
{{block "checkStats" .}} 
{{- else }}
{{block "longRunningCheckStats" .}} 
{{- end }}
{{- end }}

```

Reply:
I think it creates issues with indentation